assets/runtime/functions

```diff
@@ -145,6 +145,13 @@ redmine_finalize_database_parameters() {
     DB_USER=${DB_USER:-${POSTGRESQL_ENV_USER}}
     DB_PASS=${DB_PASS:-${POSTGRESQL_ENV_PASS}}
     DB_NAME=${DB_NAME:-${POSTGRESQL_ENV_DB}}
+
+    # fall back using sqlite3
+  else
```

Comment:
I don't think this does what you expected it to do.  

The above two if statements are designed to sort of autoconfigure the image if attached to a mysql or postgres container.  I think this kind of auto-linking of environment variables has been deprecated in docker-compose but regardless.  Your echo statement is going to always output even when configured for mysql/postgres.